packages/reusable-blocks/src/components/reusable-blocks-menu-items/reusable-block-convert-button.js

```diff
@@ -176,6 +176,7 @@ export default function ReusableBlockConvertButton( {
 								placeholder={ __( 'My pattern' ) }
 							/>
 							<ToggleControl
+								__nextHasNoMarginBottom
```

Comment:
As far as I can tell, this package is not used in the app code anymore. And the term "reusable block" has been [changed](https://github.com/WordPress/gutenberg/pull/63514) to "synced patterns". But I think we can assume this create pattern modal is the same as the previous two.